examples/new-dawn/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@playwright/test": "1.44.0",
-    "@synthetixio/synpress": "4.0.0-alpha.9",
+    "@synthetixio/synpress": "workspace:*",
```

Comment:
is this expected?

Reply:
expected and needed:  https://turbo.build/repo/docs/crafting-your-repository/structuring-a-repository#specifying-packages-in-a-monorepo